docs/cloud/manage/backups/export-backups-to-own-cloud-account.md

```diff
@@ -88,8 +88,8 @@ SETTINGS base_backup = S3('https://testchbackups.s3.amazonaws.com/backups/<base-
 ### Restore from a backup {#restore-from-a-backup}
 
 ```sql
-RESTORE DATABASE test_backups 
-AS test_backups_restored 
```

Comment:
this is restoring `test_backups` (backed up above) AS a new database, this doesnt seem incorrect to me